src/hearings/utils/case-flags.utils.ts

```diff
@@ -92,7 +92,7 @@ export class CaseFlagsUtils {
       const partyName = party.partyName ? party.partyName : (foundPartyFromService ? foundPartyFromService.partyName : '');
       const reasonableAdjustments: string[] = party.individualDetails?.reasonableAdjustments ? party.individualDetails.reasonableAdjustments : [];
       const allFlagsId: string[] = reasonableAdjustments.slice();
-      if (party.individualDetails && party.individualDetails.interpreterLanguage) {
+      if (party.individualDetails?.interpreterLanguage) {
```

Comment:
Should this be 
```suggestion
      if (party?.individualDetails?.interpreterLanguage) {
```

Reply:
@udaydenduluri33 - the `party` will always have a value, and the `individualDetails` property is optional and will be present only for PartyType `IND`.